backend/src/src-bdpimport/src/test/java/com/bosch/osmi/sw360/bdp/datasink/thrift/ThriftUploaderTest.java

```diff
@@ -140,7 +141,7 @@ public void testIfIsAlreadyInDbWrapperEmpty() {
 
     @Test
     public void testIfIsAlreadyInDbWrapperEmptyList() {
-        Optional<String> toCheck = thriftUploader.searchExistingEntityId(Optional.of(Collections.EMPTY_LIST),
```

Comment:
The Collections.emptyList() hast the element type T (new Style, since Java 1.5), the Collections.EMPTY_LIST has no element type (old Style since Java 1.2).
The Eclipse compiler cannot detect the correct type of Collections.EMPTY_LIST (and eclipse always recommend the style).
At runtime i expect no differences.